HLTrigger/JetMET/plugins/HLTScoutingCaloProducer.cc

```diff
@@ -0,0 +1,167 @@
+// -*- C++ -*-
+//
+// Package:    HLTrigger/JetMET
+// Class:      HLTScoutingCaloProducer
+// 
+/**\class HLTScoutingCaloProducer HLTScoutingCaloProducer.cc HLTrigger/JetMET/plugins/HLTScoutingCaloProducer.cc
+
+Description: Producer for ScoutingCaloJets from reco::CaloJet objects
+
+*/
+//
+// Original Author:  Dustin James Anderson
+//         Created:  Fri, 12 Jun 2015 15:49:20 GMT
+//
+//
+
+// system include files
+#include <memory>
+
+// user include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/EDProducer.h"
```

Comment:
I think this must be:

#include "FWCore/Framework/interface/stream/EDProducer.h"